[2022-09-17 14:50:03,104] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: xcom_dag.t1 manual__2022-09-17T14:50:01.523854+00:00 [queued]>
[2022-09-17 14:50:03,131] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: xcom_dag.t1 manual__2022-09-17T14:50:01.523854+00:00 [queued]>
[2022-09-17 14:50:03,131] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-09-17 14:50:03,131] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-09-17 14:50:03,131] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-09-17 14:50:03,198] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): t1> on 2022-09-17 14:50:01.523854+00:00
[2022-09-17 14:50:03,207] {standard_task_runner.py:52} INFO - Started process 1949 to run task
[2022-09-17 14:50:03,223] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'xcom_dag', 't1', 'manual__2022-09-17T14:50:01.523854+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/xcom_dag.py', '--cfg-path', '/tmp/tmpuhuv7w7b', '--error-file', '/tmp/tmpmpesnl81']
[2022-09-17 14:50:03,224] {standard_task_runner.py:80} INFO - Job 22: Subtask t1
[2022-09-17 14:50:03,333] {task_command.py:369} INFO - Running <TaskInstance: xcom_dag.t1 manual__2022-09-17T14:50:01.523854+00:00 [running]> on host 3f1973d04161
[2022-09-17 14:50:03,459] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=xcom_dag
AIRFLOW_CTX_TASK_ID=t1
AIRFLOW_CTX_EXECUTION_DATE=2022-09-17T14:50:01.523854+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-17T14:50:01.523854+00:00
[2022-09-17 14:50:03,482] {python.py:173} INFO - Done. Returned value was: None
[2022-09-17 14:50:03,503] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=xcom_dag, task_id=t1, execution_date=20220917T145001, start_date=20220917T145003, end_date=20220917T145003
[2022-09-17 14:50:03,547] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-17 14:50:03,700] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
